slides/common/cicd.md

```diff
@@ -0,0 +1,37 @@
+name: What about CI/CD for orchestration?
```

Comment:
Do you need the `name:` here?

(If so, no problem, but I would suggest a shorter name, to generate an easier permalink!)

Reply:
i was dumb and didn't understand the use